app/config/defaults.yml

```diff
@@ -1,6 +1,7 @@
 parameters:
     packagist_host: ~
     packagist_metadata_dir: "%kernel.cache_dir%/composer-packages-build"
+    packagist_dumper_compress: 0
```

Comment:
Read description of property in class maybe and then let me know if you still feel that way?